src/gui/dialogs/DialogStateful.cpp

```diff
@@ -2,6 +2,11 @@
 #include "DialogRadioButton.hpp"
 #include "gui.h"
 
+constexpr uint8_t PROGRESS_BAR_X_PAD = 10;
```

Comment:
should be
`constexpr static const` or `constexpr static` (equal in this case)

Reply:
constexpr const == constexpr
I will add static.